doc/style.css

```diff
@@ -53,3 +53,7 @@ div.chapter {
 p {
     margin-top: 0;
 }
+
+code.literal {
+    white-space: nowrap;
```

Comment:
What is this for?

Reply:
It keeps the label part of the `[horizontal]` labeled list from being broken into more than one line. Before adding `code.literal { white-space: nowrap;}`, to `style.css`, the new entry in the manual looks like this:

<img width="453" alt="image" src="https://github.com/user-attachments/assets/68bcb1fc-6748-431b-b959-399e0cc8e76a">

...and after adding that bit of CSS, it looks like this:

<img width="442" alt="image" src="https://github.com/user-attachments/assets/69624a1d-b9aa-491f-b8ba-488dbea69cd0">

Reply:
update: for the next push, i've changed this bit of CSS to:
```CSS
table.horizontal tbody tr td:nth-child(1) {
    white-space: nowrap;
    padding-right: 1.5em;
}
```

this makes it so that:
- the `nowrap` property value is now applied only to the left column of a `[horizontal]` labeled list (which fixes an issue where other instances of `code.literal` were prevented from wrapping in the middle of a paragraph, which led to some awkwardly-short lines leading up to the then-unbreakable code), and 
- there is now some padding on the right (which is missing in the 2nd screenshot above).

Reply:
might want to consider making this a separate commit

Reply:
done!